### DISCUSSION TITLE
useServerSeoMeta with Pinia and composable Request

### DISCUSSION THREAD

**Discussion by mrFANRA** [nuxt/nuxt#23584] [Questions] [answered]
Hi! Help me please.

Im write string to pinia  in onRequest method inside composable Fetch. After this calling useServerSeoMeta, but html title tag without this string.

In log and templates this string exists.

https://stackblitz.com/edit/nuxt-starter-f1t3t8?file=app.vue

**Comment by TheAlexLichter** [comment, marked as answer]
Some suggestions:

1. Use `script setup`
2. use useServerSeoMeta after your fetch call

Try:

```
<template>
  Store:
  <pre>
  {{ title }}
  </pre>
</template>

<script setup>
import { useTempStore } from './store/tempStore.js'
const tempStore = useTempStore();
const title = computed(() => tempStore.getTitle);

await useMyFetch('https://www.colourlovers.com/api/colors/new?format=json');

console.log('im app log: ', title.value);

useServerSeoMeta({
  title: title.value,
});
</script>
```

![image](https://github.com/nuxt/nuxt/assets/640208/a869f184-bceb-4e26-8396-a446b2d2a2c8)

**Reply by mrFANRA** [reply]
Damn... thank you very much!
